raml2markdown/src/acl-api/acl-api.raml

```diff
@@ -91,12 +100,12 @@ description: |
         Authorization: !include includes/properties/authorization-headers.yaml
       responses:
         204:
-        403:
+        401:
           body:
-            example: !include includes/examples/crud_403.json
-        404:
+            example: !include includes/examples/unauthorized-401.json
```

Comment:
just wondering current file naming: unauthorized-401, crud-403, crud-404.
why not: crud-401, crud-403, crud-404
or: unauthorized-401, forbidden-403, not-found-404